src/common/middleware/logger/logger.middleware.ts

```diff
@@ -0,0 +1,17 @@
+export function httpObserveLogger(req, res, next) {
```

Comment:
@types/expressにRequest, Response, NextFunctionの型が存在するので使いたいですね。
```suggestion
import { NextFunction, Request, Response } from "express";

export function httpObserveLogger(
  req: Request,
  res: Response,
  next: NextFunction,
) {

```

Reply:
対応しました!

0ea1a84c57699cf99246a11343efc7e1017b2557